lib/har-to-curl.js

```diff
@@ -76,7 +76,14 @@ harToCurl.fromEntry = function(entry) {
 	}).join('');
 
 	if (entry.request.postData) {
-		command += ' -d "' + entry.request.postData.text + '"';
+		var text = entry.request.postData.text;
+		if ( text === '') {
+			// TODO: support param.fileName, param.contentType, param.comment
+			text = entry.request.postData.params.map(function(param) {
+				return param.name + '=' + param.value ;
```

Comment:
I admit name and value must be urlencoded.

Reply:
Firefox v96.0.2: values are in raw form;
Chrome WebInspector v537.36: values are urlencoded.